boards/st/steval_stwinbx1/steval_stwinbx1.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: steval_stwinbx1
+name: STEVAL STWINBX1 Development kit
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+ram: 786
+flash: 2048
+supported:
+  - adc
+  - backup_sram
+  - counter
+  - gpio
+  - i2c
+  - pwm
+  - spi
+  - usb_device
+  - watchdog
```

Comment:
This list is only used to build a twister test campaign. At least some of these are only available internally and then can't be used for test purpose (i2c, spi) so they don't need to be added here. Likely usb_device and backup_sram could be removed as well.

Reply:
done